src/App.tsx

```diff
@@ -1,15 +1,30 @@
-import React from 'react';
+import React, { useContext, useEffect } from 'react';
 import 'bulma/bulma.sass';
 import '@fortawesome/fontawesome-free/css/all.css';
 import './App.scss';
 
-import classNames from 'classnames';
+// import classNames from 'classnames';
 import { PostsList } from './components/PostsList';
-import { PostDetails } from './components/PostDetails';
+// import { PostDetails } from './components/PostDetails';
```

Comment:
still not resolved
![image](https://github.com/mate-academy/react_dynamic-list-of-posts/assets/55200530/04e923c1-4708-4ab3-8fbd-86a8c511794e)